templates/endpoint_policies/s3_endpoint_policy.json.tpl

```diff
@@ -0,0 +1,115 @@
+{
+  "Statement": [
+    {
+      "Sid": "restrict-access-dirty-bucket-download",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::tdr-upload-files-cloudfront-dirty-intg/*",
+      "Condition": {
+        "ArnEquals": {
+          "aws:PrincipalArn": [
+            "arn:aws:iam::${account_id}:role/TDRDownloadFilesRole",
```

Comment:
Do these roles need the environment adding?

Reply:
They don't, that's how they're named because naming things consistently is not something we do :smile: